tool/src/js/type_generation/mod.rs

```diff
@@ -382,10 +382,14 @@ impl<'ctx, 'tcx> TyGenContext<'ctx, 'tcx> {
             Some(SpecialMethod::Iterable) => "[Symbol.iterator]".to_string(),
             Some(SpecialMethod::Iterator) => "#iteratorNext".to_string(),
 
-            _ if method.param_self.is_none() => {
+            None if method.param_self.is_none() => {
                 format!("static {}", self.formatter.fmt_method_name(method))
             }
-            _ => self.formatter.fmt_method_name(method),
+            None => self.formatter.fmt_method_name(method),
+            _ => panic!(
```

Comment:
this shouldn't be reverted

Reply:
Why not? Unknown attributes should not be making it into backends.

Reply:
I think in this PR it shouldn't panic, once you have auto it can because it'll be unreachable. Moot point I guess